docs/administration/high-availability/design/octopus-for-high-availability-on-aws.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Designing Octopus HA on AWS
+description: Information on configuring Octopus High Availability hosted in AWS.
+position: 30
+---
+
+This section will walk through the different options and considerations for the components required when setting up Octopus High Availability in [AWS](https://aws.amazon.com/).
+
+:::hint
+If you are setting up Octopus High Availability on Azure or on-premises please see the following guides:
+- [Azure](/docs/administration/high-availability/design/octopus-for-high-availability-on-azure.md)
+- [On-Premises](/docs/administration/high-availability/design/octopus-for-high-availability-on-premises.md)
+:::
+
+## Setting up AWS for Octopus: High Availability 
+
+For the sake of simplicity, this guide assumes that all of the servers used for your Octopus High Availability instance are hosted in AWS.
+
+:::warning
+If you are choosing [IaaS](https://en.wikipedia.org/wiki/Infrastructure_as_a_service) on AWS then the [On-Premises](/docs/administration/high-availability/design/octopus-for-high-availability-on-premises.md) guide might be a better approach for you as you may have your Domain Controllers, SQL Database Server and load balancers in the cloud.
+:::
+
+### Compute
+
+To install Octopus nodes you will need at least to machines running Windows Server 2016 +. There’s only one choice when building virtual machines in AWS, and that’s [EC2 Instances](https://aws.amazon.com/ec2/instance-types/). There are a number of different instance types to choose from. When selecting the size of the instance, We generally find sticking with the General purpose size is the best option. Choosing the rights specs for the instance depends on how many nodes you plan to use.
+
+### Database
+
+Each Octopus Server node stores project, environment and deployment-related data in a shared Microsoft SQL Server Database. Since this database is shared, it's important that the database server is also highly available. 
+
+If you don't have a SQL cluster in AWS then AWS provides a fully managed and highly available sql database as a service called [Amazon RDS for SQL Server](https://aws.amazon.com/rds/sqlserver/).
+
+Choosing a SQL edition and size is dependent on your organization and Octopus usage. The great thing about using AWS RDS is that you can start small and scale on demand as your Octopus usage grows. For further information see our [SQL Server Database](/docs/installation/sql-server-database.md) page, which explains the editions and versions of SQL Server that Octopus supports and explains the requirements for how the database must be configured.
```

Comment:
Scaling the size of course is an option, but I'm not sure that changing the edition is possible after you've created it. A little Googling led me to this: https://stackoverflow.com/a/60302666/15507446

Even the SQL Server Database installation page doesn't offer many considerations about which edition should be used. Does it make sense to offer a suggestion here? I think in the past I've suggested customers use Standard Edition.

Reply:
Added recommendation, and re-worded slightly